spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Admin::UsersController, type: :controller do
+  login_admin
+  describe 'batch actions' do
+    let(:users) { create_list(:user, 3) }
+
+    it 'collection batch action' do
+      user_ids = users.map(&:id)
+      expect(User).to receive(:destroy_collection) do |arg|
+        expect(arg.map(&:id)).to match_array user_ids
+      end
+      allow(User).to receive_message_chain(:batch_actions, :include?).with(:destroy_collection) {true}
+      allow(User).to receive(:destroy_collection)
```

Comment:
Is expect not enoght to define message stub?
05.08.2015 20:05 пользователь "sktocha" <notifications@github.com> написал:

> In spec/controllers/users_controller_spec.rb
> <https://github.com/leschenko/ab_admin/pull/26#discussion_r36326360>:
>
> > @@ -0,0 +1,19 @@
> > +require 'spec_helper'
> > +
> > +describe Admin::UsersController, type: :controller do
> > +  login_admin
> > +  describe 'batch actions' do
> > +    let(:users) { create_list(:user, 3) }
> > +
> > +    it 'collection batch action' do
> > +      user_ids = users.map(&:id)
> > +      expect(User).to receive(:destroy_collection) do |arg|
> > +        expect(arg.map(&:id)).to match_array user_ids
> > +      end
> > +      allow(User).to receive_message_chain(:batch_actions, :include?).with(:destroy_collection) {true}
> > +      allow(User).to receive(:destroy_collection)
>
> Because default ab_admin user model don't have collection batch action,
> yes, but expect on args is important.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/leschenko/ab_admin/pull/26/files#r36326360>.
>